routes/api_v2.php

```diff
@@ -40,5 +40,9 @@
                 Route::get('csv', ['middleware' => 'auth.user', 'uses' => 'OAuth2SummitEventsApiController@getAllVoteablePresentationsV2CSV']);
             });
         });
+        // ticket types
+        Route::group(['prefix' => 'ticket-types'], function () {
+            Route::get('', 'OAuth2SummitsTicketTypesApiController@getAllBySummitV2');
```

Comment:
u need to set auth middleware
['middleware' => 'auth.user', ....

Reply:
done